### PR TITLE
Fix: Cursor pointer issue in permission checkbox settings icon

### PR DESCRIPTION
**Pull request by pandeyprafull**
### What does it do?
Fixes Inconsistent cursor state for clickable elements

### Why is it needed?
To fix issue [20768](https://github.com/strapi/strapi/issues/20768) : Inconsistent cursor state for clickable elements

### How to test it?

1. Go to main menu setting.
2. Then click on Users & Permissions plugin > Roles
3. Click on any role.
4. Now if we open one of any accordion in the list and hover over the icon beside checkbox.
5. You will be able get the cursor pointer on hover. 

### Related issue(s)/PR(s)
fixes [20768](https://github.com/strapi/strapi/issues/20768)

### CODE REVIEW

File: packages/core/admin/admin/src/pages/Settings/pages/Roles/ListPage.tsx [hunk 1]
```diff
@@ -173,7 +173,7 @@ const ListPage = () => {
             rowCount={rowCount}
             footer={
               canCreate ? (
-                <TFooter onClick={handleNewRoleClick} icon={<Plus />}>
+                <TFooter cursor={'pointer'} onClick={handleNewRoleClick} icon={<Plus />}>
```
Comment (markkaylor) [inline packages/core/admin/admin/src/pages/Settings/pages/Roles/ListPage.tsx:176]: Could you pleases always pass prop strings directly 🙏 

```suggestion
                <TFooter cursor='pointer' onClick={handleNewRoleClick} icon={<Plus />}>
```

File: packages/core/admin/admin/src/pages/Settings/pages/Roles/ListPage.tsx [hunk 2]
```diff
@@ -221,6 +221,7 @@ const ListPage = () => {
             <Tbody>
               {roles?.map((role, index) => (
                 <RoleRow
+                  cursor={'pointer'}
```
Comment (markkaylor) [inline packages/core/admin/admin/src/pages/Settings/pages/Roles/ListPage.tsx:224]:
```suggestion
                  cursor='pointer'
```

**Comment by pandeyprafull** [comment]
@markkaylor Will keep in mind the requested format. For now I have commited your suggested changes, Please make it process the PR for furthur approvals

**Comment by markkaylor** [comment]
It looks you are using the wrong quotes as well for props in the components

See lint/prettier error:

```
    176:33  error  Replace `'pointer'` with `"pointer"`  prettier/prettier
    224:26  error  Replace `'pointer'` with `"pointer"`  prettier/prettier
```

I think you should be able to fix these errors with:

```
yarn lint:fix && yarn prettier:write
```

**Comment by pandeyprafull** [comment]
okay @markkaylor , checking.

**Comment by pandeyprafull** [comment]
@markkaylor can you please merge the  PR all required checks passed right now

**Review by markkaylor — APPROVED** [review]

**Pull request merged**